Assets/Tests/InputSystem/CoreTests_Actions.cs

```diff
@@ -29,7 +29,41 @@
 // in terms of complexity.
 partial class CoreTests
 {
-    #if UNITY_EDITOR
+    [Test]
+    [Category("Actions")]
+    public void Actions_WhenBindingsAreResolved_BindingIndexPointsToFirstBindingInAction()
+    {
+        InputSystem.AddDevice<Gamepad>();
+
+        var map = new InputActionMap("map");
+        map.AddAction("action2", binding: "<Gamepad>/buttonNorth");
+        var action1 = map.AddAction("action1", binding: "<Gamepad>/leftStick/x", processors: "normalize(min=-1,max=1,zero=-1)");
+        action1.AddBinding("<Gamepad>/rightStick/x", processors: "normalize(min=0,max=1)");
+
+        map.Enable();
+
+        Assert.That(action1.ReadValue<float>(), Is.EqualTo(0.5f));
+    }
+
+    [Test]
+    [Category("Actions")]
+    public void Actions_WhenResetActionState_BindingIndexPointsToFirstBindingInAction()
```

Comment:
The binding index aspect seems like implementation detail where the connection to the test contents isn't quite clear.

Reply:
Yeah, I felt the same. I couldn't see a clearer way to test it though. It's a roundabout way of testing the binding index by using the processors attached to different bindings, which is horribly brittle in itself. Can you think of a better way?

Reply:
I think the test contents are fine. Just that the names seem misleading. Maybe `Actions_ReadingValueAfterInitialResolve_AppliesProcessorsFromFirstBinding` or something.